3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/utils/ConversionUtils.scala

```diff
@@ -339,6 +339,31 @@ object ConversionUtils {
     }
 
   /**
+   * Convert a String to a String containing a
+   * Redshift-compatible Double.
+   *
+   * @param str The String which we hope contains
+   *        a Double
+   * @param field The name of the field we are
+   *        validating. To use in our error message
+   * @return a Scalaz Validation, being either
+   *         a Failure String or a Success Double
+   */
+  def stringToDouble(field: String,str: String):Validation[String, Double] = {
```

Comment:
Change this definition to:

`stringToMaybeDouble: Validation[String, Option[Double]]`, and update your code accordingly...